.github/workflows/stubtest-unused-whitelist.yml

```diff
@@ -25,11 +25,8 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      # Keep the mypy version in sync with what we use for stubtest in tests.yml
       - name: Install dependencies
-        run: |
-          python -m pip install -U pip
-          pip install -U git+git://github.com/python/mypy@master
+        run: pip install -U $(grep mypy== requirements-tests-py3.txt)
```

Comment:
Nit: `-U`/`--upgrade` is not necessary, since we specify the exact version to install.